litellm/llms/bedrock_httpx.py

```diff
@@ -53,7 +53,9 @@
     ChatCompletionToolCallFunctionChunk,
     ChatCompletionDeltaChunk,
 )
+from litellm.caching import DualCache
 
+iam_cache = DualCache()
```

Comment:
switch to in memory cache

Reply:
InMemoryCache - https://github.com/BerriAI/litellm/blob/fb96f07ccb28fffbfe7bd9e9723d48c16c4bb581/litellm/caching.py#L59